src/java.xml/share/classes/javax/xml/parsers/SAXParser.java

```diff
@@ -25,6 +25,8 @@
 
 package javax.xml.parsers;
 
+import java.lang.SuppressWarnings;
```

Comment:
So,
Basically every instance of where we have used `@SuppressedWarnings` we can leave these as they are not causing any harm.